TODO.md

```diff
@@ -9,24 +9,31 @@
 - Mechanistic Interpretability > partially complete
 - Alignment
 - Value Learning
-- Corrigibility
-- Optimization
-- Takeoff
-- Timelines
-- P/doom
-- Deception
-- Agent Foundations
-- Scalable Oversight
-- Robustness
-- Evaluations (evals)
-- Adversarial training
-- Superalignment
-- Constitutional AI
-- Natural abstractions
+- Corrigibility > partially complete
```

Comment:
Is this entire PR documenting future changes? Because right now in the published website there is nothing about corrigibility, right?